src/main/java/com/thealgorithms/stacks/StackUsingQueue.java

```diff
@@ -0,0 +1,83 @@
+package com.thealgorithms.stacks;
+
+import java.util.LinkedList;
+import java.util.Queue;
+
+// Class to implement stack operations using queues
+class StackUsingQueue {
+
+    // Two queues to manage the stack operations
+    Queue<Object> queue1;
+    Queue<Object> queue2;
+
+    // Constructor initializes the queues
+    public StackUsingQueue() {
+        queue1 = new LinkedList<Object>();
+        queue2 = new LinkedList<Object>();
+    }
+
+    // Recursive function to manage the content of the queues
+    public void manageQueue(Queue<Object> queue1, Queue<Object> queue2) {
+        // Base case: if queue1 is empty, return
+        if (queue1.isEmpty()) {
+            return;
+        }
+        // Remove an element from queue1
+        Object val = queue1.remove();
+        // Recursive call for the rest of the elements in queue1
+        manageQueue(queue1, queue2);
+        // Add the removed element to queue2
+        queue2.add(val);
+    }
+
+    // Function to add an element to the stack, which is implemented by queue1
+    public void push(Object x) {
+        queue1.add(x);
+    }
+
+    // Function to remove the top element from the stack
+    public Object pop() {
```

Comment:
Exception handling for the pop function if no elements is present is required.

Reply:
Hi @susanhsrestha,
i added the Exception handling.
Please check.

Reply:
LGTM!